pkg/kritis/metadata/metadata_test.go

```diff
@@ -0,0 +1,263 @@
+				{
+					AffectedLocation: &vulnerability.VulnerabilityLocation{},
+					FixedLocation: &vulnerability.VulnerabilityLocation{
+						Version: &pkg.Version{
+							Kind: pkg.Version_NORMAL,
+						},
+					},
+				},
+			},
+			true,
+		},
+		{"fix version maximum is not fixable",
+			[]*vulnerability.PackageIssue{
+				{
+					AffectedLocation: &vulnerability.VulnerabilityLocation{},
+					FixedLocation: &vulnerability.VulnerabilityLocation{
+						Version: &pkg.Version{
+							Kind: pkg.Version_MAXIMUM,
+						},
+					},
+				},
+			},
+			false,
+		},
+		{"fix location nil is not fixable",
+			[]*vulnerability.PackageIssue{
+				{
+					AffectedLocation: &vulnerability.VulnerabilityLocation{},
+					FixedLocation:    nil,
+				},
+			},
+			false,
+		},
+		{"one issue fixable one issue not fixable is not fixable",
+			[]*vulnerability.PackageIssue{
+				{
+					AffectedLocation: &vulnerability.VulnerabilityLocation{},
+					FixedLocation: &vulnerability.VulnerabilityLocation{
+						Version: &pkg.Version{
+							Kind: pkg.Version_NORMAL,
+						},
+					},
+				},
+				{
+					AffectedLocation: &vulnerability.VulnerabilityLocation{},
+					FixedLocation:    nil,
+				},
+			},
+			false,
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			actualFixable := IsFixAvailable(tc.pis)
+			if actualFixable != tc.expectedFixable {
+				t.Fatalf("Expected \n%v\nGot \n%v", tc.expectedFixable, actualFixable)
+			}
+		})
+	}
+}
+
+func makeRawAttestationPgp(signature, id string) RawAttestation {
+	return RawAttestation{
+		SignatureType: PgpSignatureType,
+		Signature: RawSignature{
+			Signature:   signature,
+			PublicKeyId: id,
+		},
+		SerializedPayload: []byte{},
+	}
+}
+
+func makeOccAttestationPgp(signature, id string) attestation.Attestation {
+	return attestation.Attestation{
+		Signature: &attestation.Attestation_PgpSignedAttestation{
+			PgpSignedAttestation: &attestation.PgpSignedAttestation{
+				Signature: signature,
+				KeyId: &attestation.PgpSignedAttestation_PgpKeyId{
+					PgpKeyId: id,
+				},
+			},
+		},
+	}
+}
+
+func makeRawAttestationGeneric(sig, id string, payload string) RawAttestation {
+	return RawAttestation{
+		SignatureType:     GenericSignatureType,
+		SerializedPayload: []byte(payload),
+		Signature: RawSignature{
+			PublicKeyId: id,
+			Signature:   sig,
+		},
+	}
+}
+
+func makeOccAttestationGeneric(sigs, ids []string, payload string) attestation.Attestation {
+	signatures := []*common.Signature{}
+	for i, sig := range sigs {
+		newSig := &common.Signature{
+			PublicKeyId: ids[i],
+			Signature:   []byte(sig),
+		}
+		signatures = append(signatures, newSig)
+	}
+	return attestation.Attestation{
+		Signature: &attestation.Attestation_GenericSignedAttestation{
+			GenericSignedAttestation: &attestation.GenericSignedAttestation{
+				SerializedPayload: []byte(payload),
+				Signatures:        signatures,
+			},
+		},
+	}
+}
+
+func strCompare(a, b string) bool { return a > b }
```

Comment:
Moving it right next to where it's used? or you can just define the function in the parameter.

Reply:
Done. I defined it in the parameter